src/pages/learn/whats-new/index.tsx

```diff
@@ -0,0 +1,138 @@
+import groq from 'groq'
```

Comment:
this particular page can be a root `/new` page

no need to put it in `/learn` since it's a "fixture"

I like "learn" when it reads like a sentence.

thing of `new` and a root itself that could have sub-pages